stable/traefik/README.md

```diff
@@ -18,8 +18,6 @@ resources _cluster-wide_.
 ## Prerequisites
 
 - Kubernetes 1.4+ with Beta APIs enabled
-- You are deploying the chart to a cluster with a cloud provider capable of provisioning an
-external load balancer (e.g. AWS or GKE)
```

Comment:
I know the idea of managing this with a daemonset technically eliminates this prerequisite, but since the whole daemonset thing is sort of a "use at your own risk" option that I really still don't endorse, I would rather this prereq weren't struck out. I would suggest putting it back in _or_ amending it to say this prereq can be ignored if and only if you're willing to adopt the naive approach of scaling Traefik via daemonset.